Packages/Tracking/Core/Runtime/Scripts/UltraleapSettings.cs

```diff
@@ -0,0 +1,73 @@
+
+using System;
+using System.Collections.Generic;
+
+using UnityEngine;
+
+using UnityEditor;
+
+namespace Leap.Unity
+{
+
+    [CustomEditor(typeof(UltraleapSettings))]
+    public class UltraleapSettingsDrawer : Editor
+    {
+
+        public override void OnInspectorGUI()
+        {
+            base.OnInspectorGUI();
+
+            LeapSubSystemSection();
+        }
+
+        private void LeapSubSystemSection()
+        {
+            EditorGUILayout.Space(10);
+            EditorGUILayout.BeginVertical();
+            EditorGUILayout.LabelField("Leap XRHands Subsystem", EditorStyles.boldLabel);
+            
+            var readOnlyText = new GUIStyle(EditorStyles.textField);
+            readOnlyText.wordWrap = true;
+            EditorGUILayout.HelpBox("The Leap subsystem should be used when not using OpenXR for hand tracking inputs." +
+                "\n \n This cannot be toggled during runtime.", MessageType.Info, true); ;
+            
+            var settingsTarget = target as UltraleapSettings;
+            bool editorBool = settingsTarget.LeapSubsystemEnabled;
+            EditorGUILayout.BeginHorizontal();
+            settingsTarget.LeapSubsystemEnabled = EditorGUILayout.ToggleLeft("Enable the leap subsystem ", editorBool);
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.EndVertical();
+        }
+    }
+
+
+    //[CreateAssetMenu(menuName = "Ultraleap/Settings/PluginSettings")]  /// comment out so the user cannot create these but may be needed for development
```

Comment:
no longer needed?

Reply:
So this is the only way to create a new SO. Not needed for the user but might be useful if we ever need to delete the scriptable object during development this is the only way to get it back. 

Tempted to leave it in commented out for this reason

Reply:
I think if we were to do that, we should just have an option to reset and some cached defaults - ideally users shouldn't be creating/deleting this asset